src/send_command.rs

```diff
@@ -614,7 +614,7 @@ enum AuthenticateActivity {
     WaitingForAuthenticateResponse,
     /// Waiting until the client flow user provides the authenticate data.
     ///
-    /// Specifically, [`ClientFlow::set_authenticate_data`].
+    /// Specifically, [`SendCommandState::set_authenticate_data`].
```

Comment:
`ClientFlow` is correct, I'm not sure why I don't get a warning, because `ClientFlow `it's not imported? (See comment.)